app/src/configs/env.ts

```diff
@@ -31,6 +32,9 @@ export const ENABLE_PLANS = setEnv('REACT_APP_ENABLE_PLANS', 'false') === 'true'
 /** Activate the teams menu */
 export const ENABLE_TEAMS = setEnv('REACT_APP_ENABLE_TEAMS', 'false') === 'true';
 
+/** Activate the teams menu */
+export const ENABLE_HEALTHCARE = setEnv('REACT_APP_ENABLE_TEAMS', 'false') === 'true';
```

Comment:
why do we have 2 envs for the healthcare module

Reply:
Nice catch Thanks 🙌